mbof/serializers.py

```diff
@@ -14,12 +14,12 @@ class Meta:
         fields = ('url', 'loginName', 'displayName', 'aboutMe', 'reputation',)
 
 
-class MessageSerializer(serializers.HyperlinkedModelSerializer):
+class EventSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
-        model = Message
+        model = Event
         fields = ('url', 'messageText', 'latitude', 'longitude', 'altitudeMeters', 'owner', 'postingTime', 'startTime',
```

Comment:
The `messageText` above and other occurrences of `message` in this PR makes me think you need to finish the changes to #5 and pull it into this branch before completing this PR.